Header header
uint8 robot_num
geometry_msgs/Polygon[6] footprints
